packages/manager/apps/pci-ai-notebooks/src/pages/notebooks/create/Create.page.tsx

```diff
@@ -0,0 +1,70 @@
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router-dom';
+import BreadcrumbItem from '@/components/breadcrumb/BreadcrumbItem.component';
+import Guides from '@/components/guides/Guides.component';
+import { useGetCatalog } from '@/hooks/api/catalog/useGetCatalog.hook';
+import OrderFunnel from './_components/OrderFunnel.component';
+import { useGetRegions } from '@/hooks/api/ai/capabilities/useGetRegions.hook';
+import { useGetFramework } from '@/hooks/api/ai/notebook/capabilities/useGetFramework.hook';
+import { useGetEditor } from '@/hooks/api/ai/notebook/capabilities/useGetEditor.hook';
+import { useGetSshkey } from '@/hooks/api/sshkey/useGetSshkey.hook';
+import { mockedSuggestion } from '@/__tests__/helpers/mocks/suggestion';
+
+export function breadcrumb() {
+  return (
+    <BreadcrumbItem
+      translationKey={`breadcrumb`}
+      namespace="pci-ai-notebooks/notebooks/create"
+    />
+  );
+}
+
+const Notebook = () => {
+  const { t } = useTranslation('pci-ai-notebooks/notebooks/create');
+  const { projectId } = useParams();
+  /* /
+  const suggestionsQuery = useGetSuggestions(projectId, {
+    refetchOnWindowFocus: false,
+  });
+  */
```

Comment:
Instead of commenting the future api call to use the mock, I'd prepare the hook and api layer for the suggestion endpoint, and return the mock from the api layer. By doing so, we will only have to change the api layer when the endpoint is working in the backend